esp/src/webpack.config.js

```diff
@@ -95,7 +95,11 @@ module.exports = function (env) {
         resolve: {
             alias: {
                 "clipboard": path.resolve(__dirname, "node_modules/clipboard/dist/clipboard")
-            }
+            },
+            // WebPack >= v5
+            // fallback: {
```

Comment:
Would this need to be toggled back whenever someone wanted to test out changes to hpcc-js?